modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/object/validation/rule/CommerceReturnReturnableItemsObjectValidationRuleEngineImpl.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.commerce.internal.object.validation.rule;
+
+import com.liferay.object.constants.ObjectValidationRuleConstants;
+import com.liferay.object.scope.ObjectDefinitionScoped;
+import com.liferay.object.validation.rule.ObjectValidationRuleEngine;
+import com.liferay.portal.kernel.language.Language;
+import com.liferay.portal.kernel.util.HashMapBuilder;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Crescenzo Rega
+ */
+@Component(service = ObjectValidationRuleEngine.class)
+public class CommerceReturnReturnableItemsObjectValidationRuleEngineImpl
+	implements ObjectDefinitionScoped, ObjectValidationRuleEngine {
+
+	public static final String RETURNABLE_ITEMS_VALIDATION_RULE_ENGINE =
+		"commerce-return-returnable-items-validation-rule-engine";
+
+	@Override
+	public Map<String, Object> execute(
+		Map<String, Object> inputObjects, String script) {
+
+		Map<String, Object> results = HashMapBuilder.<String, Object>put(
+			"validationCriteriaMet", true
+		).build();
+
+		// TODO This check must be done on the returned quantities
```

Comment:
remove the class if we are not ready to implement it
add it again when we have all we need